libs/game/tilemap.ts

```diff
@@ -509,6 +509,21 @@ namespace tiles {
         return scene.tileMap.getTileImage(tiles.getTileLocation(col, row).tileSet);
     }
 
+    /**
+     * Returns true if the tile at the given location is the same as the given tile;
+     * otherwise returns false
+     * @param location
+     * @param tile
+     */
+    //% blockId=maplocationistile block="tile at $location is $tile"
+    //% location.shadow=mapgettile
+    //% tile.shadow=tileset_tile_picker tile.decompileIndirectFixedInstances=true
+    //% blockNamespace="scene" group="Collisions" blockGap=8
+    export function tileAtLocationEquals(location: Location, tile: Image) {
+        const scene = game.currentScene();
+        return location && tile && location.tileSet === scene.tileMap.getImageType(tile);
```

Comment:
would it be a situation where this block can get called and there's no scene.tileMap?

Reply:
Good catch, should match the behavior of others and ignore invalid calls; fixed ~